src/commonMain/kotlin/exchange.dydx.abacus/state/model/TradingStateMachine.kt

```diff
@@ -1264,7 +1293,7 @@ open class TradingStateMachine(
                 val modifiedFills = fills?.toIMutableMap() ?: mutableMapOf()
                 var subaccountFills = fills?.get(subaccountText)
                 if (staticTyping) {
-                    val newFills = internalState.wallet.account.subaccounts?.get(subaccountNumber)?.fills?.toIList()
```

Comment:
was this piece of internal state already released? 
just want to make sure we're isolating state changes